src/config/detekt/detekt.yml

```diff
@@ -15,6 +15,7 @@ complexity:
     ignoreDeprecated: true
     ignorePrivate: true
     ignoreOverridden: false
+    excludes: ['**/HintrAPIClient.kt']
```

Comment:
This linter exclusion can be temporary since we'll be able to remove the old `calibrate` method eventually